tatu_re/model/linear_regression.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+from sklearn.preprocessing import PolynomialFeatures
+from sklearn.impute import SimpleImputer
+from sklearn.pipeline import Pipeline
+from sklearn.linear_model import LinearRegression
+
+from tatu_re.model.data_processing import IndicatorsTransformer
+
+def linear_regression_pipeline():
+
+    pipe = Pipeline([
```

Comment:
Pipeline currently not working

IndexError: single positional indexer is out-of-bounds